src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VcfLineMapper.java

```diff
@@ -33,12 +36,20 @@
     private final VariantVcfFactory factory;
 
     public VcfLineMapper(VariantSource source) {
+        if (!VariantSource.Aggregation.NONE.equals(source.getAggregation())) {
```

Comment:
the issue here is a bit different, the straightforward message here would be:
> " should be used to read genotyped VCFs only, but the VariantSource.Aggregation is set to BASIC/EXAC/EVS"
but I'm not sure if genotyped is understood as the same as non-aggregated.

what about:
> `" should not be used to read aggregated VCFs, but the VariantSource.Aggregation set to " + source.getAggregation().toString());`
?